content/elections/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "OHBM OSSIR 2021 elections"
```

Comment:
"elections" is tricky verbiage as we're not putting it to a popular vote this year 😬. Maybe swap "elections" for "committee" or "officers"? I think the student & postdoc SIG still refers to the process of selecting new officers as "elections", though, and doesn't actually vote either. Just a thought.

Reply:
@62442katieb: Good point! I think this is very much up to you to decide. Let us know what you prefer.